src/main/java/io/cdap/plugin/gcp/bigtable/sink/BigtableSink.java

```diff
@@ -69,7 +69,6 @@
 @Name(BigtableSink.NAME)
 @Description("This sink writes data to Google Cloud Bigtable. " +
   "Cloud Bigtable is Google's NoSQL Big Data database service.")
-@Requirements(capabilities = "bypass_cmek_check")
```

Comment:
is this supposed to be removed?

Reply:
Yes it is intended. Since the Bigtable sink does not create non-CMEK compliant resources, it should not have the bypass_cmek_check requirement.